examples/bela/bela_start_scsynth.scd

```diff
@@ -16,4 +14,8 @@ s.options.blockSize = 16;
 s.options.numInputBusChannels = 2;
 s.options.numOutputBusChannels = 2;
 
+// Uncomment these lines to enable access from remote hosts
+// s.options.bindAddress = "0.0.0.0"
```

Comment:
The purpose of this script is to start `scsynth` locally so that one can connect from it remotely, therefore I would leave these lines uncommented by default. It's still good to leave a note that these are the ones that allow remote login.

Reply:
Agree, my main point was to put a reminder for us there, since bindAddress is required now, and it wasn't in previous sc versions